concert/experiments/addons.py

```diff
@@ -388,6 +388,7 @@ async def _check_timestamp(self, producer):
         self.timestamp_missing = False
         i = 0
         last_acquisition = await self._experiment.acquisitions[-1].get_state() == "running"
```

Comment:
I know this isn't new but it's really a hack, we should improve this in the future.